### PR TITLE
ref(ts) Convert Gravatar to typescript

### PR DESCRIPTION
**Pull request by markstory**

### CODE REVIEW

File: src/sentry/static/sentry/app/components/avatar/gravatar.tsx [hunk 1]
```diff
@@ -4,28 +4,35 @@ import qs from 'query-string';
...
+  gravatarId?: string;
+  placeholder?: string;
+  /**
+   * Should avatar be round instead of a square
+   */
+  round?: boolean;
+};
+
+type State = {
+  MD5?: any;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/avatar/gravatar.tsx]: Is there a `@types/crypto-js` we could import and type this as?

Comment (billyvg) [reply src/sentry/static/sentry/app/components/avatar/gravatar.tsx]: Or can we just type it as a Function?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/components/avatar/gravatar.tsx (hunk 1 above)
Comment (markstory) [reply src/sentry/static/sentry/app/components/avatar/gravatar.tsx]: I was having trouble importing the types and *not* importing the library.

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Pull request merged**